### PR TITLE
Added a button to jump to current date

### PR DESCRIPTION
**Pull request by Pijuli**
This adds a button to jump back to the current day. By default is hidden and the param of the component is optional defaulting to false, so no side effects should appear.

### CODE REVIEW

**Comment by Pijuli** [comment]
ng e2e is also failing me, showing some weird snapshots...
snapshots_1

**Comment by alfonsserra** [comment]
Could you please comment the following line in systelab-components/e2e/app.po.ts and commit again?

const data = await this.takeScreenshot();


We have disabled the snapshot testing because the snapshots that we get in Travis are different that the ones we get in our laptops.

For the next releases we are going to follow an alternative approach.

**Comment by vinyulis** [comment]
I suggest to to set this new property as true by default.

You are doing the pull request in an old branch. Can you do the same pull request for master?. Can you add this new property in README.md, please?.

**Comment by Pijuli** [comment]
> I suggest to to set this new property as true by default.
> 
> You are doing the pull request in an old branch. Can you do the same pull request for master?. Can you add this new property in README.md, please?.

As an advise from Aptiva team, I set it to false on purpose. That won't break old specifications for this component. But yes, I can set it to true by default if you decide to do it that way. There's no problem for us.
Regarding the questions, yes and yes :)

> Could you please comment the following line in systelab-components/e2e/app.po.ts and commit again?
> 
> const data = await this.takeScreenshot();
> 
> We have disabled the snapshot testing because the snapshots that we get in Travis are different that the ones we get in our laptops.
> 
> For the next releases we are going to follow an alternative approach.

I can, but then the next line needs this data variable. Can you provide me with some more info about it?

Thank you for the fast reply both of you 👍

**Comment by Pijuli** [comment]
@alfonsserra 
For this branch, I will proceed as in master:
return await true

File: src/app/systelab-components/datepicker/datepicker.component.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ export class Datepicker implements OnInit, AfterViewInit, DoCheck, OnDestroy {
 	@Input() public inputExpandHeight: boolean;
 	@Input() public markPreviousAfterDate = false;
 	@Input() public inputFontSize: number;
+	@Input() public showTodayButton = true;
```
Comment (alfonsserra) [inline src/app/systelab-components/datepicker/datepicker.component.ts]: Set showTodayButton to false in order to have the same behaviour than before

Comment (Pijuli) [reply src/app/systelab-components/datepicker/datepicker.component.ts]: Done

**Pull request merged**